server/controllers/profile.js

```diff
@@ -71,5 +71,22 @@ exports.uploadProfileImage = asyncHandler(async (req, res, next) => {
       .status(200)
       .send({ message: "The profile has been successfully updated" });
   });
+});
+
+// @route GET /profile/load/sitters
+// @desc Get all sitter profiles
+// @access Private
+exports.loadSitterProfiles = asyncHandler(async (req, res) => {
+  const sitterProfiles = await Profile.find({ isSitter: true });
+
+  if (!sitterProfiles) {
+    res.status(500);
+    throw new Error("An error occured");
+  }
```

Comment:
Mongoose `find` method returns an array. That means that your if block will never execute since arrays will always be truthy. So you should return the array to the client regardless of whether it's empty or not, and let the client figure out how to render content.

Reply:
changed to `!sitterProfiles.length`